src/lib/logstore/log_store_family.cpp

```diff
@@ -286,8 +294,8 @@ nlohmann::json LogStoreFamily::dump_log_store(const log_dump_req& dump_req) {
     if (dump_req.log_store == nullptr) {
         folly::SharedMutexWritePriority::ReadHolder holder(m_store_map_mtx);
         for (auto& id_logstore : m_id_logstore_map) {
-            auto store_ptr{id_logstore.second.m_log_store};
-            const std::string id{std::to_string(store_ptr->get_store_id())};
+            auto store_ptr = id_logstore.second.log_store;
```

Comment:
nit: auto const&

Reply:
My changes are only to replace { to = for better readability, but I will do as much as I can the one you pointed out in this code.